terraform/environments/data-platform/glue_job.tf

```diff
@@ -0,0 +1,118 @@
+
+locals {
+  default_arguments = {
+    "--continuous-log-logGroup" = aws_cloudwatch_log_group.log_group.name
+  }
+}
+
+
+# load the glue script to bucket as bucket object
+resource "aws_s3_object" "object" {
+  bucket=module.s3-bucket.bucket.id
```

Comment:
Would really recommend changing the name of the bucket resource to something more descriptive than s3-bucket.

Reply:
And same again for the s3_object